softwerkskammer/testIntegration/app_test.js

```diff
@@ -2,7 +2,7 @@
 var expect = require('must-dist');
 var httpRequest = require('request');
 var sinon = require('sinon').sandbox.create();
-var conf = require('../testutil/configureForTest');
+var conf = require('../testutil/configureForTestWithDB.js');
```

Comment:
Und ich hatte ja eben gemeint, dass ich das simple-configure umbauen kann, so dass wir mehrere Configs parallel haben können. So lange müsstest Du dann auch noch warten ;-) Wäre mir wichtiger als die Information zu verwässern, wo überall in den Tests die DB gebraucht wird.